.github/workflows/checks.yml

```diff
@@ -21,7 +21,11 @@
 
 name: pr checks
 
-on: pull_request
+on: 
+  pull_request:
+    paths:
+      - ./src
```

Comment:
It should to be src/** and just yarn.lock IMO
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-including-and-excluding-paths
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-including-paths

Reply:
done